inst/examples-shiny/02_text/app.R

```diff
@@ -3,13 +3,11 @@ library(bslib)
 
 # Define UI for dataset viewer app ----
 ui <- page_sidebar(
-
   # App title ----
   title = "Shiny Text",
 
   # Sidebar panel for inputs ----
   sidebar = sidebar(
-
```

Comment:
I'm confused as to why this newline got removed, but it didn't in `01_hello`?

Reply:
I did attempt to add code changes and restore the newlines manually but it was a bit of work so I bailed. So I probably introduced the difference in this line.